test/server/test-error-test.js

```diff
@@ -0,0 +1,72 @@
+var expect             = require('chai').expect;
+var buildTestError     = require('../../lib/test-error');
+var plainTextDecorator = require('../../lib/test-error/plain-text-decorator');
+var readFile           = require('../../lib/utils/read-file-relative');
+var TYPE               = require('../../lib/test-error/type');
+
+var MAX_STRING_LENGTH = 10;
+
+function compareMsg (expectedMsg, err) {
+    var msg = buildTestError(err, plainTextDecorator, MAX_STRING_LENGTH);
+
+    expect(expectedMsg).to.be.equal(msg);
+}
+
+describe('Test Error', function () {
```

Comment:
I know it's a lot of code, but it would be nice to test all types of error. So we'll be sure what we doing things right. It's also would b e nice to have a test that checks if error message has test for it.